stdlib/public/SwiftShims/SwiftStdint.h

```diff
@@ -21,6 +21,18 @@
 // Clang has been defining __INTxx_TYPE__ macros for a long time.
 // __UINTxx_TYPE__ are defined only since Clang 3.5.
 
+#if defined(_WIN32)
```

Comment:
Alternatively, if the `__INT_TYPE__` versions are a Darwin-specific workaround then they should be behind an affirmative check for Darwin. Then everyone else defaults to the cleaner stdint.h form.